guides/common/modules/con_remote-execution-workflow.adoc

```diff
@@ -1,7 +1,12 @@
 [id="remote-execution-workflow_{context}"]
 = Remote Execution Workflow
 
-When you run a remote job on hosts, for every host, {Project} performs the following actions to find a remote execution {SmartProxy} to use.
+{Project} prefers remote execution on hosts that are registered through proxy.
```

Comment:
```suggestion
{Project} prefers remote execution on hosts that are registered through a {SmartProxy}.
```

IMHO this is not true, because `Prefer registered through proxy for remote execution` defaults to "no" on Foreman 3.3.

Reply:
Shall I go with the `Smart Proxy` that you suggested?

Reply:
I still believe this is not entirely true because you want to have this change cherry-picked to 3.3 & I've checked the default behaviour on Foreman.

Reply:
Reworded that line :+1: